optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -881,6 +881,64 @@ This avoids hard coding a solver time, because the unit test might run on arbitr
 This approach ensures that the test runs long enough to find a feasible solution, even on slow machines.
 But it does not run a millisecond longer than it strictly must, even on fast machines.
 
+Additionally, if you use a `ConstraintProvider` and have `org.optaplanner:optaplanner-test` as a dependency in your pom, you can test per constraint using a
+`ConstraintVerifier`.
+
+Add the `org.optaplanner:optaplanner-test` dependency to your `pom.xml` dependencies section:
```

Comment:
This doesn't make much sense. First you say you need to have `optaplanner-test` in your POM, then you tell people to add it there.

Also, consider that we also support Gradle. Personally, I wouldn't bother with the POM example - just mention the `org.optaplanner:optaplanner-test` GAV.

Reply:
I do think it belongs here, as this is the section about testing. I think we can avoid the entire Maven/Gradle question by just mentioning the GAV.